contracts/cairo/Scarb.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.0"
 [dependencies]
 starknet = "2.3.1"
 snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.12.0" }
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git" }
```

Comment:
Please pin a specific version to avoid issues in the future

Reply:
And add in the readme that we are using OZ upgradable contracts

Reply:
youre right, done!